sdk/healthcareapis/Azure.ResourceManager.HealthcareApis/tests/Azure.ResourceManager.HealthcareApis.Tests.csproj

```diff
@@ -1,4 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <PackageReference Include="Azure.ResourceManager.EventHubs" VersionOverride ="1.1.0-beta.1"/>
```

Comment:
is there a reason that we have to add this version override?

Reply:
There is no Azure.ResourceManager.EventHubs reference version in the Packages.Data.props file and the reason why not choose 1.0.0 version is because that version will cause [Pipeline issue](https://dev.azure.com/azure-sdk/public/_build/results?buildId=2208424&view=logs&j=bb926d89-89e7-581f-c2a9-0b96e9ca8a72&t=89e0f785-6520-5689-44e6-c45b4430a324&l=2276)

Reply:
We should add it to the packages.data.props in the same location as the other arm packages used by test projects.